superset/assets/javascripts/dashboard/components/Dashboard.jsx

```diff
@@ -153,6 +157,27 @@ class Dashboard extends React.PureComponent {
     return this.props.filters[sliceId];
   }
 
+  addHistory({ isReplace = false, title }) {
```

Comment:
I'm confused about this method:
* why using destructuring in the function parameters?
* why do we need to alter the browser history?
* `title` is never passed to it, won't that lead to clearing the title?